controllers/tab/publicationEntry/PublicationEntryTabHandler.inc.php

```diff
@@ -169,6 +169,20 @@ function saveForm($args, $request) {
 				}
 			} else {
 				// Could not validate; redisplay the form.
+				// Provide entered tagit fields values
+				$tagitKeywords = $form->getData('keywords');
+				if (is_array($tagitKeywords)) {
+					$tagitFieldNames = $form->_metadataFormImplem->getTagitFieldNames();
+					$locales = array_keys($form->supportedLocales);
+					foreach ($tagitFieldNames as $tagitFieldName) {
+						foreach ($locales as $locale) {
+							$formTagitData[$locale] = array_key_exists($locale . "-$tagitFieldName", $tagitKeywords) ? $tagitKeywords[$locale . "-$tagitFieldName"] : array();
```

Comment:
On the first iteration, `$formTagitData` will not exist and OTOH there will be a PHP warning. I'd suggest initializing before the loop:

    $formTagitData = array();

...then removing the `unset`, which won't be necessary any more.